client/telecomUniverseComponents/section-back-link/index.js

```diff
@@ -0,0 +1,8 @@
+import angular from 'angular';
+
+import tucSectionBackLink from './section-back-link.component';
+
+export default angular
+  .module('tucSectionBackLink', [])
```

Comment:
Missing `ui.router`?
ref: https://github.com/ovh-ux/ovh-manager-telecom/blob/80cb5d72447eab46a5bc9ddea6313a853e882e82/client/telecomUniverseComponents/section-back-link/section-back-link.html#L12

Reply:
I've added the missing `ui.router` dependency.